File: javap/variable_scope/book.java
package javap.variable_scope;

class Book {
    int data = 50;// instance variable
    static int m = 100;// static variable

    int method() {
        int n = 90;// local variable
        return n;
    }

    public static void main(String args[]) {
        Book b1 = new Book();
        System.out.println("Static variable" + " " + Book.m);
        System.out.println("Instance variable" + " " + b1.data);
        System.out.println("Local variable" + " " + b1.method());
    }
}

File: javascriptp/variable_scope/book.js
var Book = function () {
  this.data = 50; // instance variable
  this.method = function () {
    var n = 90; // local variable
    return n;
  };
};

const newBook = new Book();
console.log(newBook);
console.log("Instance variable" + " " + newBook.data);
console.log("Local variable" + " " + newBook.method());
